### DISCUSSION TITLE
A section similar to the Rust "Are we x yet" sites

### DISCUSSION THREAD

**Discussion by elliotwaite** [mojodojodev/mojodojo.dev#44] [Ideas]
I was thinking that I like the Rust "Are we x yet" sites (https://wiki.mozilla.org/Areweyet) and that it would be nice to have something similar for Mojo. But instead of them being spread out over a bunch of different websites, perhaps they could all be in one place, on mojodojo.dev.

It would basically be a collection of open-source Mojo-related repos organized into categories and subcategories. And each grouping could perhaps have a short text description describing the current state of development of the repos in that category, such as:
* The production-readiness of the top repos.
* What can or can't be done yet in that category.
* The collective strengths and weaknesses of the repos in that category.

It could also have some additional features that I wish the Rust "Are we x yet" sites would have, such as:
* Listing more useful data about each repo and being able to sort all the repos in that section by any of those data points. Some ideas for data points are:
  * Repo name and description
  * Total stars
  * Stars gained within the last month
  * Total repo contributions
  * Repo contributions within the last month
  * Total downloads (from PyPI, or whatever is used as the Mojo package index)
  * Downloads within the last month
 
* Also, some graphs comparing all the repos in each category would be nice. For example, a graph showing each repos total stars over time and one showing their total contributions over time. This would make it easier to see which repos are popular, gaining popularity, and are being actively developed. Some good ideas for potentially helpful graphs can be found here: https://vesoft-inc.github.io/github-statistics/

It might be a little early for adding something like this, but it looks like there are already some Mojo repos starting to pop up (e.g. https://github.com/automata/mojograd). It could even just start out as a minimal version that could be improved over time. That way when people saw it on Mojo Dojo, even if it didn't have many repos listed yet, they would know that they could check back there in the future for that type of info.

**Comment by mojodojodev** [comment]
Hey yeah it’s a great idea thanks, there really isn’t anything available yet though, even mojograd isn’t functional. I will do this once the ecosystem builds up a little and useful projects start popping up

**Reply by elliotwaite** [reply]
👍 Sounds good.
